### PR TITLE
Cluster Service Version Cleanup

### PR DESCRIPTION
**Pull request by mew2057**
A collection of fixes for the `ClusterServiceVersion`:

* Added missing descriptions for the Custom Resource Fields.
* Fixed a bug in the description copying code
* Added missing fields back to the CRD.
* Fixed the X-Descriptors.

### CODE REVIEW

File: stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/0.9.1/ibm-spectrum-scale-csi-operator.v0.9.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -62,64 +62,153 @@ spec:
...
         displayName: Secret Counter
         path: secretCounter
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:number
-      - description: Mapping of K8s node with SpectrumScale node.
-        displayName: nodeMapping
-        path: nodeMapping
-        x-descriptors: []
+      - description: Node selector for SpectrumScale CSI Plugin.
+        displayName: pluginNodeSelector
```
Comment (yadaven) [inline stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/0.9.1/ibm-spectrum-scale-csi-operator.v0.9.1.clusterserviceversion.yaml]: displayName should be Plugin Node selector and path should be pluginNodeSelector ?

Comment (mew2057) [reply stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/0.9.1/ibm-spectrum-scale-csi-operator.v0.9.1.clusterserviceversion.yaml]: See Pull Request #60, that has  CSV fixes

**Comment by mew2057** [comment]
Added a playbook to automatically change the operator version for us  and make sure it's all up  to date.

**Comment by mew2057** [comment]
Interestingly the PR passed and my local branch failed CI/CD

**Comment by mew2057** [comment]
Pending this CI/CD clearing I'm going to merge so test can have a complete  RC.

**Pull request merged**